deployments/api_infra/labcas.py

```diff
@@ -0,0 +1,188 @@
+import re
+import os
+import datetime
+import json
+import shutil
+
+import requests
+
+# This should be same as in the mapped Archive in docker-compose.yaml
+# real_LabCAS_archive='/labcas-data/labcas-backend/archive/mcl'
```

Comment:
We can probably drop references to MCL, right? It's a dead project.

Reply:
@nutjob4life I guess yeah!